javascript/linked-list/trees/trees.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+class BTNode {
+  constructor(value, left = null, right = null) {
+    this.value = value;
+    this.left = left;
+    this.right = right;
+  }
+}
+
+class BinaryTree {
+  constructor(root = null) {
+    this.root = root;
+  }
+  preOrder(node) {
+    // could do anything but we're just logging
+    console.log(node.value);
+    // go left
+    if (node.left) {
+      this.preOrder(node.left);
+    }
+    // go right
+    if (node.right) {
+      this.preOrder(node.right);
+    }
+    return;
+  }
```

Comment:
The challenge asks that you return an array of values in the correct order, but you are not returning anything. Also, the initial call should require no arguments (we know it will always start its traversal from the root).